tools/codegen/api/python.py

```diff
@@ -561,13 +768,15 @@ def signature(f: NativeFunction, *, method: bool = False) -> PythonSignature:
         tensor_options_args.append(PythonArgument(
             name='dtype',
             type=BaseType(BaseTy.ScalarType),
-            default=_dtype_default_type_hack(name),
+            default=_dtype_default_type_hack(name, pyi=pyi),
```

Comment:
nit: shall we do 'if pyi' here (like other tensor option fields below) instead of pushing this flag to the 'XXX_hack' :)

Reply:
ha yeah... I figured gathering all parts of the hack into one place to blowtorch later would look nicer :) But happy to do that instead.